doc/pyproject_toml.rst

```diff
@@ -67,12 +67,18 @@ requires
             "configparser; python_version == '2.7'"
       ]
 
-dev-requires
-  Packages that are required for development. This field is in the same format
-  as ``requires``.
+extras-require
+  Lists of packages needed for every optional feature. The requirements
+  are specified in the same format as for ``requires``. The requirements of
+  the two reserved extras ``test`` and ``doc`` as well as the extra ``dev``
+  are installed by ``flit install``. For example:
+
+  .. code-block:: toml
```

Comment:
This will emit sphinx warning message: `WARNING: Pygments lexer name 'toml' is not known`.
I'm using Sphinx 1.7.4.

Reply:
hmm, we could add `pygments-github-lexers` to the `docs` extra requirements